elements/lisk-p2p/src/p2p.ts

```diff
@@ -12,7 +12,7 @@
  * Removal or modification of this copyright notice is prohibited.
  *
  */
-
+import { randomBytes } from 'crypto';
```

Comment:
I added all the code related to secret calculation from @mitsujutsu 's PR #4023  so it will be handled there, I just wanted a secret to initialize the PeerBook

Reply:
Its related to the above comment, it will come once the other #4023 will be merged and i will resolve it

Reply:
@ManuGowda @diego-G `getRandomBytes` is not exported to top level. It is only exported to other modules within `lisk-cryptography`.

Reply:
@ishantiw Leaving a note here  `lisk-cryptography:getRandomBytes` was updated to be exposed. This line was modified to use `lisk-cryptography` in netgroup PR.